services/core-web/common/actionCreators/mineActionCreator.js

```diff
@@ -357,29 +357,3 @@ export const deleteMineComment = (mineGuid, commentGuid) => (dispatch) => {
       throw new Error(err);
     });
 };
-
-export const getUserInfo = (token, errorMessage) => (dispatch) => {
```

Comment:
This whole API request was completely unnecessary (the info it's looking for is stored in the token)